README.md

```diff
@@ -76,15 +76,17 @@ Trunk-based development is the safest branching pattern. It prevents lost work,
 
 Resources for improving the delivery pipelines to move from "basic" to "elite"
 
-- [Continuous Delivery Pipelines](https://leanpub.com/cd-pipelines) by Dave Farley
+- [Continuous Delivery Pipelines](https://leanpub.com/cd-pipelines) (book) by Dave Farley
```

Comment:
now that we've added a third bit of data, we should prob turn this into a table instead of a list.  prob do like | Title | Type | Author(s) |